core/domain/services/graphql/connections/RootQueryEventsConnection.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\connections;
+
+use EEM_Event;
+use EventEspresso\core\domain\services\graphql\abstracts\AbstractRootQueryConnection;
+use EventEspresso\core\domain\services\graphql\connection_resolvers\EventConnectionResolver;
+
+class RootQueryEventsConnection extends AbstractRootQueryConnection
```

Comment:
This is not needed, it's already provided by WP GQL. So, it won't allow you duplicate the field